R/01-createDuplicateModal.R

```diff
@@ -0,0 +1,131 @@
+#' Creates a modalDialog for detectDuplicates module
+#'
+#' @param vars dataframe column names
+#' @param session current session
+createDuplicateModal <- function(vars, session) {
+  ns <- session$ns
+  modalDialog(
+    title = "Duplicate Identifier",
+    easyClose = TRUE,
+    size = "l",
+    footer = modalButton("Close (Without Saving Changes)"),
```

Comment:
Can you shift the button "Close Modal and Transfer to App", see pic.,  and rename it to e.g. "Accept (Transfer to App)"?
![image](https://user-images.githubusercontent.com/16759098/215453270-b2185f94-5994-4aac-8292-dc1acca3fe07.png)